src/Symfony/Component/RateLimiter/Tests/Storage/CacheStorageTest.php

```diff
@@ -76,4 +76,11 @@ public function testFetchNonExistingState()
 
         $this->assertNull($this->storage->fetch('test'));
     }
+
+    public function testDelete()
+    {
+        $this->pool->expects($this->once())->method('deleteItem')->with(sha1('test'))->willReturn(true);
```

Comment:
This line uses `$this->pool` which is a mock created in setUp. 

We say that we "expect" that "once" the "method `deleteItem`" is called with `sha1('test')`. We also return true. 

If what we expect does not happen, then the test will fail